react/yarn.lock

```diff
@@ -4320,6 +4327,15 @@ cross-spawn@^7.0.0:
     shebang-command "^2.0.0"
     which "^2.0.1"
 
+cross-spawn@^7.0.1:
```

Comment:
what's cross-spawn?

Reply:
It's part of the cross-env package: https://www.npmjs.com/package/cross-env